blog/2019-10-17-debezium-newsletter-02-2019.adoc

```diff
@@ -0,0 +1,68 @@
+= Debezium's Newsletter 02/2019
+ccranfor
+:awestruct-tags: [ community, news, newsletter ]
+:awestruct-layout: blog-post
+
+Welcome to the Debezium community newsletter in which we share blog posts, group discussions, as well as StackOverflow
+questions that are relevant to our user community.
+
+== Upcoming Events
+
+* https://jokerconf.com[JokerConf - Practical change data streaming uses cases with Apache Kafka and Debezium]
+* https://qconsf.com[QCon San Fransisco - Practical change data streaming use cases with Apache Kafka and Debezium]
+
+== Articles
+
+This https://debezium.io/blog/2019/10/01/audit-logs-with-change-data-capture-and-stream-processing/[recent blog] by Gunnar Morling discusses how Debezium combined with Kafka stream post processing can deliver an enriched stream of events suitable for even the most complicated of tasks like audit tracking.
```

Comment:
Could you add an introductory sentence like "There were a lot of blog posts touching on Debezium lately; here are some of the latest posts you shouldn't miss:". Then a list with the ones you already got and also these:

* https://speakerdeck.com/jbfletch/using-kafka-to-discover-events-hidden-in-your-database
* https://medium.com/convoy-tech/logs-offsets-near-real-time-elt-with-apache-kafka-snowflake-473da1e4d776
* https://info.crunchydata.com/blog/postgresql-change-data-capture-with-debezium

And then add a sentence with the link to the online-resources page.